### ISSUE TITLE
Suggestion: add a field for companies which are open to remote work

### ISSUE THREAD

**Issue by matthewcummings** [connor11528/companies-in-the-bay-area#3]

**Comment by connor11528** [comment]
Hey thanks @matthewcummings I like this. Remote work isn't always a binary thing. Sometimes teams are fully distributed, other times it's more a one or a few work from home days a week or based on seniority. It would be cool to have more info about the team makeup and distribution for companies. Leaving this open

On the employer side, when they login to https://employbl.com and sear for candidates I'd like to have filters for geography for remote ok ✅. Many Bay Area startups will relocate people that have the right skill set out here

Still need to figure out what's most helpful on the candidate side. Have you seen https://remote.com?

**Comment by matthewcummings** [comment]
Agreed @connor11528, some indication of remote options would be great.  Given that you focus on Bay area companies, my assumption would be that they wouldn't really be "fully" distributed companies that is companies which literally have no headquarters, but more likely that they are based in the Bay area and have many some/many remote employees.  Or allow local employees to work from home a few days per week.  It's definitely not binary as you mentioned.

Yes, I've seen remote.com and other similar sites like https://weworkremotely.com and https://remoteok.io.  There are a few on GitHub as well https://github.com/remoteintech/remote-jobs.

**Comment by matthewcummings** [comment]
Also, I realize that you focus on the Bay area but this type of company list would be great for other metro areas too.
